convert-llama-h5-to-gguf.py

```diff
@@ -68,18 +69,23 @@ def permute(weights: NDArray, n_head: int) -> NDArray:
 
 print("gguf: get model metadata")
 
-llm_arch    = "llama"
-head_count  = hparams["num_attention_heads"]
+llm_arch = "llama"
+hf_repo = hparams["_name_or_path"]
```

Comment:
Dont know if this is usable. Most models dont have this and in StableLM it resolves to:
` "_name_or_path": "/fsx/ckpts/3b_tok=neox_data=pilev2-recontam=p3_lower-code_bs=8m_tp=4_pp=1_init=wang-small-init/global_step84000_hf",`

Reply:
Maybe we can have an option to get this and other strings using command line parameters?